rental.rb

```diff
@@ -1,14 +1,14 @@
 require_relative './person'
 require_relative './book'
 class Rental
-  attr_reader :person, :book
+  attr_reader :person, :rentals
   attr_accessor :date
 
-  def initialize(book, person, date)
+  def initialize(rentals, person, date)
     @date = date
     @person = person
-    @book = book
+    @rentals = rentals
+    person.rentals << @book
+    book.rentals << @person
   end
```

Comment:
- [ ] It would be better thinking about it. It doesn't make sense that `Rental class` receive and store `rentals`. The information that Rental should receive is `(date, book, person)` in its initialize. 🛠️

Reply:
@ArthurGC Hey! It's a funny mistake actually, I did a full scan and changed book to rentals and I changed the instance variable aswell. But I want to ask you, since this is my 2/3 review If I would change that is everything else is correct ?

Reply:
@ArthurGC I changed to what you adressed in the commit I have just submit it

Reply:
If you follow all suggestions from your reviewer and you didn't implement issues after your changes. Everything will be okay. 😉

Reply:
Alright I just changed it to this 


```
require_relative './person'
require_relative './book'
class Rental
  attr_reader :person, :book
  attr_accessor :date

  def initialize(book, person, date)
    @date = date
    @person = person
    @book = book
    person.rentals << @book
    book.rentals << @person
  end
end

```
is that it @ArthurGC

Reply:
I'm sorry for bothering It's just that it's my last submission that's all